devsetup/edpm/embedded-services/quadlets/ceilometer_agent_compute/ceilometer_agent_compute.yaml

```diff
@@ -0,0 +1,80 @@
+---
+apiVersion: v1
+kind: Pod
+metadata:
+  annotations:
+    bind-mount-options: /var/lib/openstack/cacerts/telemetry/tls-ca-bundle.pem:z
+  creationTimestamp: "2024-11-21T04:54:58Z"
+  labels:
+    app: ceilometeragentcompute-pod
+  name: ceilometeragentcompute-pod
+spec:
+  containers:
+  - args:
+    - kolla_start
+    env:
+    - name: KOLLA_CONFIG_STRATEGY
+      value: COPY_ALWAYS
+    - name: OS_ENDPOINT_TYPE
+      value: internal
+    image: quay.io/podified-antelope-centos9/openstack-ceilometer-compute:current-podified
+    name: ceilometeragentcompute
+    securityContext:
+      runAsGroup: 42405
+      runAsUser: 42405
+      seLinuxOptions:
+        type: ceilometer_polling_t
+    volumeMounts:
```

Comment:
Need some method of handling TLS mount requirements. It could be a matter of copying this as a base and using Ansible to add the requirements, then write the output to `/etc`.

Or, we could use a OpenShift build. Render a Containerfile that inherits from our base EDPM image, Go template this file into a ConfigMap with all of the requirements based on the cloud we're deploying too. Then `bootc switch` to that image on the EDPM nodes during the deployment.